admin/app.js

```diff
@@ -303,10 +303,16 @@ app.use(function (err, req, res, next) {
   // catch CSRF errors and redirect to the previous location
   if (err.code === 'EBADCSRFTOKEN') return res.redirect('back')
 
+  const showError = req.app.get('env') === 'development' || req.user.role === 'HELPDESK'
+
+  if (showError && err.hasOwnProperty('mtcInfo')) {
+    res.locals.mtcInfo = err.mtcInfo
+  }
+
   // render the error page
   // @TODO: provide an error code and phone number? for the user to call support
   res.locals.message = 'An error occurred'
-  res.locals.error = req.app.get('env') === 'development' ? err : {}
+  res.locals.error = showError ? err : {}
```

Comment:
in dev, we still want the stack trace.  as per the criteria - preserve existing behaviour, but allow the info to be presented as an additional to what currently happens, if its provided.